localization/imu_integration/readme.md

```diff
@@ -0,0 +1,14 @@
+\page imuintegration Imu Integration
```

Comment:
can't this be a lib of the imu_augmentation? Usually we put a directory inside src and include, with the lib_name (here is imu_integration). Is there a reason this is a separate package?

Reply:
It's also used by the graph_localizer, so it's a dep of both imu_augmentation and graph_localizer.